nix/openssl_1_0_2.nix

```diff
@@ -11,9 +11,25 @@ pkgs.stdenv.mkDerivation rec {
 
   buildInputs = [ pkgs.gnumake pkgs.perl534 ];
 
-  configurePhase = ''
-    ./config -d shared -g3 -fPIC no-libunbound no-gmp no-jpake no-krb5 no-md2 no-rc5 no-rfc3779 no-sctp no-ssl-trace no-store no-zlib no-hw no-mdc2 no-seed no-idea enable-ec_nistp_64_gcc_128 no-camellia no-bf no-ripemd no-dsa no-ssl2 no-capieng -DSSL_FORBID_ENULL -DOPENSSL_NO_DTLS1 -DOPENSSL_NO_HEARTBEATS --prefix=$out
-  '';
+  configurePhase = let
+    default_options =
```

Comment:
nit: extra spaces

Reply:
This was formatted with `nixfmt`; the default_options is probably indented because the statement isn't complete, and likely wouldn't all fit on an 80 character line.